inc/Editor/DataBinding/BlockBindings.php

```diff
@@ -149,7 +149,7 @@ private static function apply_query_input_overrides( array $query_input, array $
 			$block_name
 		);
 
-		return array_merge( $query_input, $query_input_overrides );
+		return array_merge( $query_input, $overrides );
```

Comment:
I'm not sure offhand how to easily test, but, I'm assuming this was the intent. 

This one look right to you, @chriszarate?